js/App.js

```diff
@@ -11,13 +11,14 @@ import {
 
 import { theme } from '../config';
 
-import Settings from './scenes/Settings';
-import Profile from './scenes/Profile';
-import Compass from './scenes/Compass';
-import Demos from './scenes/Demos';
+import FTU from './scenes/FTU';
 import Home from './scenes/Home';
 import Item from './scenes/Item';
 import Map from './scenes/Map';
+import Profile from './scenes/Profile';
```

Comment:
I'm really missing that we cannot do a dynamic import :( :( :( would love to load all this scenes like fte, debug, etc. on demand.